openc3/lib/openc3/topics/decom_topic.rb

```diff
@@ -0,0 +1,32 @@
+# encoding: ascii-8bit
+
+# Copyright 2023 OpenC3, Inc.
+# All Rights Reserved.
+#
+# This program is free software; you can modify and/or redistribute it
+# under the terms of the GNU Affero General Public License
+# as published by the Free Software Foundation; version 3 with
+# attribution addendums as found in the LICENSE.txt
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# This file may also be used under the terms of a commercial license
+# if purchased from OpenC3, Inc.
+
+require 'openc3/topics/topic'
+
+module OpenC3
+  class DecomTopic < Topic
```

Comment:
We have CommandDecomTopic and TelemetryDecomTopic ... should this be InterfaceDecomTopic?

Reply:
Good point. I should just put this in TelemetryDecomTopic